core/src/main/java/com/coderanch/blackjack/Cards.java

```diff
@@ -53,12 +54,18 @@ private Cards() {}
     }
 
     /**
-     * Gets a standard deck of cards.
+     * Gets a standard deck of cards. A standard deck contains all combinations of rank and suit
```

Comment:
Please put each sentence on a new line. Terminate sentences with a period.

Personally I don't think this line adds anything beyond what the `@return` tag already states, and therefore it can be omitted.

I believe Liutauras' comment pertained to the `getShuffledStandardDeck()` lacking any documentation at all.